# ------------------------------------------------------------------------------
# Proyecto: 
# Materia: Implementación de Robótica Inteligente - Fuzzy
# Descripción: Parámetros de configuración para el controlador difuso del robot.
# Fecha: 6 de junio de 2025
# Alumnos:
#   - Jonathan Arles Guevara Molina  | A01710380
#   - Ezzat Alzahouri Campos         | A01710709
#   - José Ángel Huerta Ríos         | A01710607
#   - Ricardo Sierra Roa             | A01709887
# ------------------------------------------------------------------------------

ControllerFuzzy:
  ros__parameters:
    v_max: 0.05
    v_min: 0.02
    ramp_step: 0.01
    max_error: 40.0
    alpha: 0.45
    err_NL: [-60, -60, -40, -20]
    err_NS: [-40, -20, 0]
    err_Z:  [-15, -3, 3, 15]
    err_PS: [0, 20, 40]
    err_PL: [20, 40, 60, 60]
